spec.emu

```diff
@@ -90,29 +90,36 @@ contributors: Ujjwal Sharma, Younies Mahmoud
     </emu-table>
 
     <emu-clause id="sec-getunitoptions" aoid="GetUnitOptions">
-      <h1>GetUnitOptions ( _unit_, _options_, _baseStyle_, _prevStyle_ )</h1>
-      <p>The GetUnitOptions abstract operation is called with the arguments _unit_ (which must be a String), _options_ (which must be an Object), _baseStyle_ (which must be a String), and _prevStyle_ (which must be a String) and returns a Record containing the relevant options for that unit. The following steps are taken:</p>
+      <h1>
+        GetUnitOptions (
+          _unit_: a String,
+          _options_: an ECMAScript language value,
+          _baseStyle_: a String,
+          _prevStyle_: a String,
+        ): a Record with [[Style]] and [[Display]] fields
+      </h1>
+      <dl class="header">
+      </dl>
 
       <emu-alg>
-        1. Let _stylesList_ be the List &laquo; `"long"`, `"short"`, `"narrow"` &raquo;.
+        1. Let _stylesList_ be &laquo; `"long"`, `"short"`, `"narrow"` &raquo;.
         1. If _unit_ is one of `"hours"`, `"minutes"`, `"seconds"`, `"milliseconds"`, `"microseconds"`, or `"nanoseconds"`, then
-          1. Set _stylesList_ to the list-concatenation of _stylesList_ and &laquo; "numeric" &raquo;.
+          1. Append `"numeric"` to _stylesList_.
           1. If _unit_ is one of `"hours"`, `"minutes"`, or `"seconds"`, then
-            1. Set _stylesList_ to the list-concatenation of _stylesList_ and &laquo; "2-digit" &raquo;.
-        1. Let _style_ be ? GetOption(_options_, _unit_, _stylesList_, *undefined*).
+            1. Append `"2-digit"` to _stylesList_.
+        1. Let _style_ be ? GetOption(_options_, _unit_, `"string"`, _stylesList_, *undefined*).
+        1. Let _displayDefault_ be `"always"`.
         1. If _style_ is *undefined*, then
+          1. Set _displayDefault_ to `"auto"`.
           1. If _baseStyle_ is `"digital"`, then
             1. If _unit_ is one of `"hours"`, `"minutes"`, `"seconds"`, `"milliseconds"`, `"microseconds"`, or `"nanoseconds"`, then
               1. Set _style_ to `"numeric"`.
             1. Else,
               1. Set _style_ to `"narrow"`.
           1. Else,
             1. Set _style_ to _baseStyle_.
-          1. Let _displayDefault_ to `"auto"`.
-        1. Else,
-          1. Let _displayDefault_ to `"always"`.
```

Comment:
This one I'm pretty sure ecmarkup would complain about. Are you looking at the ecmarkup linting warnings? Are you using a recent version of ecmarkup?